pycytominer/operations/variance_threshold.py

```diff
@@ -51,7 +51,7 @@ def variance_threshold(
 
     # Subset dataframe
     if samples != "all":
-        population_df.query(samples, inplace=True)
+        population_df = population_df.query(samples)
```

Comment:
For each one of these modified `population_df` lines consider adding in the named parameter within the call.
```suggestion
        population_df = population_df.query(expr=samples)
```